packages/devtools_app/lib/src/screens/network/constants.dart

```diff
@@ -69,3 +65,17 @@ class NetworkEventDefaults {
   static const receive = 1;
   static const ssl = -1;
 }
+
+class NetworkEventCustomFieldKeys {
+  static const isolateId = '_isolateId';
+  static const id = '_id';
+  static const startTime = '_startTime';
+  static const events = '_events';
+}
+
+class NetworkEventCustomFieldRemappedKeys {
```

Comment:
make this an enum since the values match the const names

Reply:
done